### ISSUE TITLE
depot config does not handle percentage signs

### ISSUE THREAD

**Issue by carandraug** [MicronOxford/cockpit#464]
If a device config has a `%`1, for example:

```
[filters]
type: cockpit.devices.microscopeDevice.MicroscopeFilter
uri: PYRO:TestFilterWheel@127.0.0.1:8004
cameras: west
filters:
  0, ND 1%%
  1, ND 2%%
```

Then starting to cockpit fails with:

```
Traceback (most recent call last):
  File "/home/carandraug/.local/bin/cockpit", line 11, in <module>
    load_entry_point('cockpit', 'gui_scripts', 'cockpit')()
  File "/home/carandraug/src/cockpit/cockpit/__init__.py", line 349, in main
    config = cockpit.config.CockpitConfig(sys.argv)
  File "/home/carandraug/src/cockpit/cockpit/config.py", line 69, in __init__
    self._depot_config = DepotConfig(self['global'].getpaths('depot-files'))
  File "/home/carandraug/src/cockpit/cockpit/config.py", line 126, in __init__
    self.read(filepaths)
  File "/home/carandraug/src/cockpit/cockpit/config.py", line 148, in read
    self[new_section][k] = v
  File "/usr/lib/python3.7/configparser.py", line 1256, in __setitem__
    return self._parser.set(self._name, key, value)
  File "/usr/lib/python3.7/configparser.py", line 1198, in set
    super().set(section, option, value)
  File "/usr/lib/python3.7/configparser.py", line 893, in set
    value)
  File "/usr/lib/python3.7/configparser.py", line 402, in before_set
    "position %d" % (value, tmp_value.find('%')))
ValueError: invalid interpolation syntax in '\n0, ND 1%\n1, ND 10%' at position 8
```

This happens because of the way reading of depot configuration happens.  To error in the case of duplicated sections, each file is read with its own `ConfigParser` instance and then merged on the `DepotConfig`. The interpolation for variables happens once when the file is read the first time and a second time when the sections and values are manually added to the `DepotConfig` instance.

**Comment by carandraug** [comment]
Fixed this in 20116f7 by disabling interpolation on the final instance where the config gets merged. Added a test case for this issue too.

**Comment by carandraug** [comment]
Fixed this in 20116f7 by disabling interpolation on the final instance where the config gets merged. Added a test case for this issue too.
